### PR TITLE
Prefilled jetconfigs for templates DEV-837

### PR DESCRIPTION
**Pull request by mohsenari**
Added `jetconfig.yaml` file to templates so that jetpack create doesn't invoke questions.

@Lagoja @LucilleH I'm not sure if the jetconfig files for these templates have the correct values. Feel free to suggest changes.

### CODE REVIEW

File: api/jetconfig.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+configVersion: "1.0"
+application:
+  name: api
+environment:
+  dev:
+    flags:
+      min-replicas: 1
+      public: true
+  preview:
```
Comment (Lagoja) [inline api/jetconfig.yaml:9]: Let's remove preview from the config for now, I think this is not ready to ship until we have the CI preview environments configured

File: api/jetconfig.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+configVersion: "1.0"
+application:
+  name: api
```
Comment (Lagoja) [inline api/jetconfig.yaml:3]: Per discussion on Slack, we should make sure that the application name is customizable at `create` time

**Review by Lagoja — APPROVED** [review]
Left 2 notes. Provided that they're resolved, this LGTM

File: api/jetconfig.yaml (hunk 2 above)
Comment (LucilleH) [reply api/jetconfig.yaml:3]: I think we can still leave the name here, but change the init chaining logic such that we only ask the first question

**Comment by mohsenari** [comment]
Merging this since further changes need to happen on the CLI side

**Pull request merged**